server/db/migrations/create-admins-table.js

```diff
@@ -0,0 +1,27 @@
+const tableName = 'admins'
```

Comment:
You should generate this file automatically with `npx knex migrate:make create-admins-table`. This will generate the migration with a timestamp like the other files. You can just copy and paste the contents into the new migration.

That command is not referenced anywhere in the project so don't worry about not having known about it 😄